src/payment/payment-methods.mock.ts

```diff
@@ -411,6 +412,13 @@ export function getPaymentMethodsMeta() {
     };
 }
 
+export function getPaymentMethodsWith3ds(): PaymentMethod[] {
```

Comment:
Could you please rename this function to: `getPaymentMethodWith3dsMock()` since all functions in this file are mocks 🍹